src/main/java/com/raisetecch/handling/demo/UserService.java

```diff
@@ -25,15 +24,25 @@ public User findUser(int id) {
             }
         }
 
+        public User entryUser(String name) {
+            if (name == null || name.isEmpty()) {
+                throw new ResourceNotFoundException("Name must not be empty or null.");
+            }
+            User user = new User(name);
+            user.setName(name);
```

Comment:
コンストラクタの仕組みの理解が必要そうです。
```suggestion
            User user = new User(name);
```
setterがなくてもコンストラクタの引数にnameをわたしているのでuserインスタンスには名前が正しく設定されます。